src/Tools/dotnet-dump/Commands/SetClrPathCommand.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Repl;
+using System.CommandLine;
+
+namespace Microsoft.Diagnostics.Tools.Dump
+{
+    [Command(Name = "setclrpath", Platform = CommandPlatform.Linux | CommandPlatform.OSX, Help = "Set the path to load coreclr DAC/DBI files.")]
+    public class SetClrPath: CommandBase
+    {
+        public AnalyzeContext AnalyzeContext { get; set; }
+
+        [Argument(Name = "clrpath", Help = "Runtime directory path.")]
+        public string Argument { get; set; }
+
+        public override void Invoke()
+        {
+            if (Argument == null)
+            {
+                WriteLine("Load path for DAC/DBI: '{0}'", AnalyzeContext.RuntimeModuleDirectory ?? "<none>");
```

Comment:
Is it normal that a set command will get the value if no argument is given? if it's consistent with other commands, that's ok

Reply:
That is the pattern for a few other SOS commands (maybe it isn't great) including the lldbplugin setclrpath that this command is patterned after.